symplyphysics/core/symbols/symbols.py

```diff
@@ -27,23 +27,24 @@ def display_name(self) -> str:
         return self._display_name
 
 
-class DimensionSymbolNew:
+class HasDimension:
     _dimension: Dimension
-    _display_name: str
-    _display_latex: str
 
-    def __init__(self,
-        display_name: str,
-        dimension: Dimension = Dimension(S.One),
-        *,
-        display_latex: Optional[str] = None) -> None:
+    def __init__(self, dimension: Dimension = Dimension(S.One)) -> None:
         self._dimension = dimension
-        self._display_name = display_name
-        self._display_latex = display_latex or self._display_name
 
     @property
     def dimension(self) -> Dimension:
         return self._dimension
+    
+
+class HasDisplay:
```

Comment:
This looks better but I don't like naming. Class should be implemented in an object, so it should describe class of objects. I suggest something like `DisplayWithLatex`